pixi_browse/tui/widgets.py

```
@@ -786,6 +818,425 @@ def on_key(self, event: Key) -> None:
+        margin-top: 1;
+        color: $text-muted;
+    }
+    """
+
+    BINDINGS = [
+        Binding("tab", "next_section", show=False, priority=True),
+        Binding("shift+tab", "previous_section", show=False, priority=True),
+        Binding("x", "swap_sides", show=False),
+        Binding("escape", "back", show=False),
+        Binding("q", "quit", show=False),
+    ]
```

Ruff (0.15.6)

[warning] Mutable default value for class attribute

(RUF012)